### PR TITLE
Clean up legacy JWT code in Pilot

### PR DESCRIPTION
**Pull request by howardjohn**
For https://github.com/istio/istio/issues/17053

### CODE REVIEW

**Comment by howardjohn** [comment]
@esnible @yangminzhu do we want to keep the protos around for istioctl?

File: pilot/pkg/security/authn/v1alpha1/policy_applier.go [hunk 1]
```diff
@@ -173,49 +172,6 @@ func convertToEnvoyJwtConfig(policyJwts []*authn_v1alpha1.Jwt) *envoy_jwt.JwtAut
...
-			},
-		}
-
-		ret.Rules = append(ret.Rules, jwt)
-	}
-	return ret
-}
-
 // ConvertPolicyToJwtConfig converts policy into Jwt filter config for envoy.
 // Returns nil if there is no JWT policy. Returns the Istio JWT filter config if USE_ENVOY_JWT_FILTER
```
Comment (yangminzhu) [inline pilot/pkg/security/authn/v1alpha1/policy_applier.go]: Also update the comments.

**Comment by howardjohn** [comment]
> I think you can remove it for now, the JWT column in the istioctl is not really useful now because it only supports the old Istio jwt filter.

Do we want it to support envoy jwt filter or should I just remove the column?

**Comment by yangminzhu** [comment]
> > I think you can remove it for now, the JWT column in the istioctl is not really useful now because it only supports the old Istio jwt filter.
> 
> Do we want it to support envoy jwt filter or should I just remove the column?

The whole command needs some re-design so I think it's okay to just remove the column, we may have a very different replacement or even remove the whole command.

**Comment by howardjohn** [comment]
Ok,I removed the column

File: pilot/pkg/security/authn/v1alpha1/policy_applier.go (hunk 1 above)
Comment (yangminzhu) [reply pilot/pkg/security/authn/v1alpha1/policy_applier.go]: cc @howardjohn

Comment (howardjohn) [reply pilot/pkg/security/authn/v1alpha1/policy_applier.go]: sorry forgot that, ill clean it up

**Pull request merged**